Update frmBootstrap.cs

JohnLaTwC opened the pull request:
Fix null de-ref exception on exit. Fix UAC shield icon.

Neo23x0 commented:
The update-shield option in the menu needs that icon, too. 
I couldn't used the once registered scheduled task and had to revert to starting the elevated RulesSync.

JohnLaTwC commented:
I updated the PR.  I think this change got lost earlier on.  I changed the update rules option to launch the scheduled task instead of invoking the updater EXE directly.  Since you set the task to run elevated, it should remove the need for UAC consent dialog and hence no longer require the shield.
